testing_utils_test.go

```diff
@@ -165,6 +152,52 @@ func newFakeMarathonEndpoint(t *testing.T, configs *ConfigContainer) *endpoint {
 	}
 }
 
+// basicAuthMiddleware handles basic auth
+func basicAuthMiddleware(config *configContainer, next http.HandlerFunc) func(http.ResponseWriter, *http.Request) {
+	unauthorized := "{\"message\": \"invalid username or password\"}"
```

Comment:
We can get rid of the escaped quotes and increase readability by wrapping the string in back ticks (`` ` ``).